src/org/scratchOrgSettingsGenerator.ts

```diff
@@ -186,9 +205,22 @@ export default class SettingsGenerator {
   private allBusinessProcesses: string[] = [];
   private readonly shapeDirName: string;
   private readonly packageFilePath: string;
-
-  public constructor(options?: { mdApiTmpDir?: string; shapeDirName?: string; asDirectory?: boolean }) {
+  private readonly capitalizeRecordTypes: boolean;
+
+  public constructor(options?: {
+    mdApiTmpDir?: string;
+    shapeDirName?: string;
+    asDirectory?: boolean;
+    capitalizeRecordTypes?: boolean;
+  }) {
     this.logger = Logger.childFromRoot('SettingsGenerator');
+    if (options?.capitalizeRecordTypes === undefined) {
+      const messages = Messages.loadMessages('@salesforce/core', 'scratchOrgSettingsGenerator');
+      void Lifecycle.getInstance().emitWarning(messages.getMessage('noCapitalizeRecordTypeConfigVar'));
```

Comment:
Should we add a test for this case?

Reply:
updated the NUT I added here to catch the warning:
https://github.com/salesforcecli/plugin-org/pull/935/commits/af99c2378630bc7b4797b61c0957694c3ba406b9#diff-6bfc3a41539529fe4f97ec0aedeff7510865079193ac0aaade4d6c87ff33fd70R102

note that those try to run `sf config unset org-capitalize-record-types` which isn't available yet so will fail.

I updated the PR desc. to explain how it should work.